docs/concepts/Configure-Command.md

```diff
@@ -0,0 +1,100 @@
+
+**Profile selection process:**
+ * If the user had setup both `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables, CLI prompts the user to confirm using environment profile. If user opts to go with environment variables, the    corresponding aws profile is chose as `__AWS_CREDENTIALS_IN_ENVIRONMENT_VARIABLE__`. Else,
+ * If no previous aws profiles are found, `ask-cli-default` is chosen as the default profile name.
+ * If multiple profile(s) are found, CLI prompts user to choose an existing profile or create a new profile.
+   * If user chooses an existing profile, CLI maps the corresponding AWS profile with ASK profile and updates the `.ask/cli_config` file and wraps up the setup process.
+   * If user chooses to create a new AWS profile, user is prompted to enter a profile name. `ask-cli-default` is chosen as default profile name to store AWS developer credentials, unless specifically mentioned by the user.
+
+**Profile setup process**
+ * If there are existing profiles, user is presented with an option to select an existing profile or choose to create new profile. If user selects an existing profile, since the corresponding `aws_access_key_id` and `aws_secret_access_key` is already available, CLI finishes the setup process with a confirmation message.
+ * If user chooses to create new profile, CLI prompts the user for a name and proceeds with the AWS profile setup process. Similar to ASK profile setup, AWS profile setup can be operated in two modes, `ask configure` and `ask configure --no-browser`. In `--no-browser` mode, CLI displays a URL which points to AWS IAM role creation page and prompts the user to open the URL in a browser. Else, CLI opens the URL in system default browser without any user intervention.
+   * User is presented with a login screen to AWS developer account. Create a new profile if you do not have one [here](https://console.aws.amazon.com/).
+   * Upon successful login, if the generated IAM user name (`ask-cli-<aws_profile_name>`) is unique, user is taken to the review page with the minimum required policies applied to create an IAM user role.
+   * If there exists an IAM role with the same name as the IAM user name, the user is redirected to the page where the user name can be updated. Clicking `next` in the `set permissions` and `Add tags` pages, user is taken to the review page. **The policies used in either of the workflows can be updated by the user if required.**
+
+Upon successful completion of the creation of the IAM user, CLI expects the user to enter the generated `aws_access_key_id` and `aws_secret_access_key` into the command prompt. The policies used in the creation of IAM role are located [here](need to add a link here). For more information on IAM roles and policies, refer to the documentation [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html).
```

Comment:
Let's first describe what are the IAM roles we are using, since we don't have a link for it for now.